docs/sources/determine-use-case/index.md

```diff
@@ -11,32 +11,52 @@ weight: 400
 
 After you’ve identified the problem process or service, you can filter and explore using labels and flame graphs to view lower levels.
+With capabilities like the Flame graph AI interpreter or the GitHub integration, Explore Profiles helps you locate the root cause and how to address it.
+
+## Use case 1: Investigate an issue
 
-## Use case 1: Know the problem service, not the cause
+Profiling data is idea when you know there is a specific service or area where there is a performance issue.
+Maybe an alert from spike in CPU led you to profiles or maybe your logs showed OOM issue for a particular service and you need to debug it.
+Explore Profiles lets you quickly drill into a service and identify the performance issue in these scenarios.
 
-If you know the affected service (use case 1), your investigation starts by viewing that service using the **Services** view and reviewing the profiles for that service.
+### Example: Know the problem service, not the cause
+
+If you know the affected service (use case 1), your investigation starts by viewing that service using the **All services** view and reviewing the profiles for that service.
 In the first scenario, you may have identified that the `checkoutservice` has an issue.
-You can select `checkoutservice` and then use the **Profile types** view to examine all profiles for that service.
+You can use **Search services** to locate `checkoutservice` and then use the **Profile types** view to examine all profiles for that service.
 
-## Use case 2: Know there is an issue, need to investigate
+### Example: Know there is an issue, need to investigate
 
 If you only know there is an issue and have to investigate, then your investigation starts by using the **All services** exploration type.
 Using the **Profile type** selector, you can check the services’ CPU processes, memory allocation, blocks, locks, exceptions, and other available profile types.
 
-After you locate the profile with a spike, select either **Profiles** to change to **Profile types** view and examine all profile types for that service, or select **Labels** to view the labels (such as `hostname` or `span_name`) for that service.
+After you locate the profile with a spike, select either **Profile types** view to examine all profile types for that service, or select **Labels** to view the labels (such as `hostname` or `span_name`) for that service.
+
+## Use case 2: Research performance and resource bottlenecks
+
+Illuminate performance issues across a number of dimensions when you are doing a proactive analysis.
+This could be a cost-cutting exercise or attempt to improve latency or memory usage in a non-incident scenario. This exercise could be across multiple services or within a service across profile types, labels, or a combination of the two.
+The flow guides you through identifying the bottleneck.
```

Comment:
Which flow?

Reply:
Let's remove the last sentence in that paragraph. We'll eventually write another use case to cover the cost-cutting scenario, but not right now. 
```suggestion
```